fixture-model-registry/code/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "experiments-get-started",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "postCreateCommand": "pip3 install --user mlem"
```

Comment:
looks like mlem is part of the requirements already, should we just install all of them (including GTO)?

Reply:
done